### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by angeldanielboscan**

### CODE REVIEW

**Comment by luiselizondo** [comment]
This doesn't solve the main problem commented at https://github.com/iiiepe/docker-nginx-drupal/issues/1#issuecomment-219452024. It just removes the mail support (which I was planning anyway) and the apt-get clean, but at the same time, you didn't remove /tmp and other temporary files which should not be necessary.

I already worked on a fix that does the same things, I'm still testing and I'll publish this week.

**Comment by angeldanielboscan** [comment]
I made a pull request by mistake my friend, no intended to solve any issue yet. 

But now that you're on the thread, It will be good to have a glimpse on that fix so I can test and assist myself

File: Dockerfile [hunk 1]
```diff
@@ -8,22 +8,41 @@ RUN locale-gen en_US.UTF-8
...
+php5-redis \
+php5-memcache \
+memcached \
+nginx-extras \
+git \
+supervisor \
+ca-certificates \
+openssl \
+sendmail --no-install-recommends \
+&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```
Comment (angeldanielboscan) [inline Dockerfile:39]: Removal of unnecessary files

**Comment by luiselizondo** [comment]
Basically you need to upgrade to Docker 1.11 to have this issue, you can read about the problem at: https://github.com/docker-library/php/issues/74 and the solution is to use `clear_env = no` on the PHP configuration, the problem is that the PHP version included with Ubuntu 14.04 does not support the clean_env opiton, so you need to upgrade PHP somehow. I'm away from the computer with the commit but I'll post it by this afternoon.

**Comment by angeldanielboscan** [comment]
The removal of unnecessary files is in [line 40](https://github.com/iiiepe/docker-nginx-drupal/pull/3/commits/d7879411b076f129ccb8945d3b0519ebcd3e8755)

**Pull request closed without merging**